nginx/metadata.csv

```diff
@@ -180,4 +180,11 @@ nginx.upstream.peers.sent_count,count,,byte,,The total amount of data sent to th
 nginx.upstream.peers.unavail,gauge,,,,How many times the server became unavailable for client requests (state "unavail") due to the number of unsuccessful attempts reaching the max_fails threshold.,0,nginx,upstr peers unavail
 nginx.upstream.peers.unavail_count,count,,,,How many times the server became unavailable for client requests (state "unavail") due to the number of unsuccessful attempts reaching the max_fails threshold (shown as count).,0,nginx,upstr peers unavail
 nginx.upstream.peers.weight,gauge,,,,Weight of the server.,0,nginx,upstr peers weight
+nginx.stream.zone_sync.status.nodes_online,gauge,,,,The number of peers this node is connected to.
+nginx.stream.zone_sync.status.msgs_in,gauge,,,,The number of messages received by this node.
+nginx.stream.zone_sync.status.msgs_out,gauge,,,,The number of messages sent by this node.
+nginx.stream.zone_sync.status.bytes_in,gauge,,,,The number of bytes received by this node.
+nginx.stream.zone_sync.status.bytes_out,gauge,,,,The number of bytes sent by this node.
+nginx.stream.zone_sync.zones.$zone_name.records_total,gauge,,,,The total number of records stored in the shared memory zone.
```

Comment:
do they have that $ in the metric name?

Reply:
If that's the case they will need some processing:
> Can only contain ASCII alphanumerics, underscores, and periods. Other characters are converted to underscores

https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/

For example, we could convert `zone_name` into a tag

Reply:
Yeah, it'll need to be parsed, as nginx can have multiple zones defined here.

Reply:
My apologies here, I completely overlooked TAGGED_KEYS, I think this is what needs to be done in this context.